app/models/user.rb

```diff
@@ -26,6 +26,10 @@ def self.from_omniauth(auth)
     role, daac = Cmr.get_role_and_daac(auth.uid, auth.credentials['access_token'])
     user.role = role
     user.daac = daac if daac
+    auth.info['first_name'] = '' if auth.info['first_name'].nil?
+    auth.info['last_name'] = '' if auth.info['last_name'].nil?
```

Comment:
`auth.info` is what's coming back from URS via omniauth, right? I don't think you should be changing what is coming back from that, it might be better to just declare new variables for first and last names.

Reply:
Also, first and last names are required fields when you register with URS. I'm not sure you need these checks in that case, especially if you are using interpolation within double quotes, and then doing a `strip`.